packages/sdk/src/controllers/ConnectorController.ts

```diff
@@ -226,6 +226,9 @@ class ConnectorConfigurator {
      * By defining the mappings, we can trigger re-download of assets (dynamic asset provider)
      * or populate filters for the query endpoint. The mapped data will be available using
      * the context parameter in the connector implementation code.
+     *
+     * for variables is should follow the format "var.[variable id]". e.g. var.6B29FC40-CA47-1067-B31D-00DD010662DA

```

Comment:
```suggestion
     * For variables it should follow the format "var.[variable id]". e.g. var.6B29FC40-CA47-1067-B31D-00DD010662DA
```

If you add an example for variables, you should probably add one for strings.

Reply:
Maybe also add that a `name` is being mapped to a `value` (this is the actual model) and that your _"var..."_ is the `value`.